src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -133,6 +138,7 @@ deliverClone( string const & name, Group * const parent ) const
   newConstitutiveRelation.m_p1Index = m_p1Index;
   newConstitutiveRelation.m_p2Index = m_p2Index;
 
+  newConstitutiveRelation.setIsClone( true );
```

Comment:
can this go inside `deliverClone` ?

Reply:
That would indeed be better, this way, set `setIsClone()` could be private.